libsel4camkes/include/camkes/dataport_caps.h

```diff
@@ -60,6 +64,16 @@ static inline size_t dataport_get_size(dataport_caps_handle_t *d)
     return d->get_size();
 }
 
+static inline size_t dataport_get_page_size(dataport_caps_handle_t *d)
```

Comment:
Do we need this function, seems it's just a convenience thing. Having just `dataport_get_page_size_bits()` seems sufficient, as the size can be derived from this easily.

Reply:
Good point, removed this.